config/vufind/RecordDataFormatter.ini

```diff
@@ -0,0 +1,45 @@
+; This file can be used to set global default options for fields, to add new fields and to configure fields
+; in the RecordDataFormatter.
+
+; This section is used to set global default option.
+[Global]
+; Enable or disable all fields by default (default is true)
+;enabled = false
+; Use change default renderMethod (default is "Simple")
+;renderType = "CombineAlt"
+; Change the default separator for array values (default is <br />)
+;separator = "; "
+; One can also add any other option.
```

Comment:
It might be worth while to document a link to the [wiki page](https://vufind.org/wiki/development:architecture:record_data_formatter) here for reference.

Reply:
Good idea. Included it.